src/config/settings.ts

```diff
@@ -322,6 +322,11 @@ const settings: Settings = {
         ? (<string>process.env.BLOCKCHAINS_ETHEREUM_PROVIDER_URL).split(',')
         : [],
     },
+    rootstock: {
+      providerUrl: process.env.BLOCKCHAINS_ROOTSTOCK_PROVIDER_URL
```

Comment:
our pattern is `${chain}_BLOCKCHAIN_PROVIDER_URL`, let's use this.

Reply:
btw, we already have setup to rootstock, so I think we can reuse it, no need for new setup, check `BlockchainRepository` and `BlockchainRepository`

Reply:
I do not remember now :) search for `BLOCKCHAIN_PROVIDER_URL` in a code

Reply:
Done